api/src/test/java/org/openmrs/api/ObsServiceTest.java

```diff
@@ -1973,4 +1974,29 @@ public void shouldNotAutomaticallySetStatusWhenManuallyCopyingAnObs() throws Exc
 		assertThat(existing.getVoided(), is(true));
 		assertThat(newObs.getStatus(), is(Obs.Status.FINAL));
 	}
+
```

Comment:
Once you move the logic to ObsValidator, you need to move the test to ObsValidatorTest

Reply:
Ok... Thanks